src/keylessBackup/encryption.ts

```diff
@@ -0,0 +1,91 @@
+import crypto from 'crypto'
+import { ec as EC } from 'elliptic'
```

Comment:
Heads up that the `elliptic` library has some _really_ nasty sharp edges. In particular it's `sign` method expects a hex-encoded string or array message _hash_ instead the message itself. What is really aweful though is that if you pass it a string that is not hex, it will simply drop the invalid hex characters, then truncate to 32-bytes and pretend it's a buffer. I would strongly recommend replacing it's  usage.

Instead I would recommend noble crypto's implementation of k256 (or ed25519 if you are not bound to k256) https://paulmillr.com/noble/